components/camel-rabbitmq/src/test/java/org/apache/camel/component/rabbitmq/RabbitMQSendDynamicAwareTest.java

```diff
@@ -0,0 +1,41 @@
+package org.apache.camel.component.rabbitmq;
```

Comment:
You need to add the ASF license header in top of these new files. Look at one of the existing java source files and copy its header.